package.json

```diff
@@ -43,6 +43,7 @@
     "redux": "^3.6.0",
     "redux-observable": "^0.12.2",
     "rxjs": "^5.0.1",
+    "sort-by": "^1.2.0",
```

Comment:
Could use lodash so the extra dep isn't needed but nbd; https://lodash.com/docs/4.17.4#sortBy